### PR TITLE
Add readme table

### PR DESCRIPTION
**Pull request by cruiseliu**

### CODE REVIEW

**Review by ultmaster — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -34,7 +34,165 @@ NNI automates feature engineering, neural architecture search, hyperparameter tu
...
+</tr>
+<tr valign="top">
+<td align="center" valign="middle">
+<b>Algorithms</b>
+</td>
+<td>
+<ul>
+<li><b>Exhaustive search</b></li>
+<ul>
+<li><a href="https://nni.readthedocs.io/en/doc-refactor/reference/hpo.html#nni.algorithms.hpo.gridsearch_tuner.GridSearchTuner">Grid Search</a></li>
```
Comment (matluster) [inline README.md:62]: not doc-refactor

Comment (cruiseliu) [reply README.md:62]: Should we use latest or stable? 🤔

Comment (matluster) [reply README.md]: Not sure. I used stable for other links in readme. But master might contain un-released features. In that case, we won't have stable.

Comment (cruiseliu) [reply README.md]: I meant, stable are dead links until next release.

**Review by J-shang — APPROVED** [review]

**Pull request merged**